server/sonar-server/src/main/java/org/sonar/server/platform/web/PlatformServletContextListener.java

```diff
@@ -44,11 +43,13 @@ public void contextInitialized(ServletContextEvent event) {
       Platform.getInstance().init(props, servletContext);
       Platform.getInstance().doStart();
       event.getServletContext().setAttribute(STARTED_ATTRIBUTE, Boolean.TRUE);
-
+    } catch (org.sonar.api.utils.MessageException | org.sonar.process.MessageException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)